File: Tareas/c++/factorial/factorial.cpp


#include <iostream>

using namespace std;

int main()
{
    int numAfactorial;
    int resultado = 1;
    cout<<"Te devolveremos el factorial de un numero!"<<endl;
    cout<<"ingresa el numero del cual quieres el factorial"<<endl;
    cin>>numAfactorial;
    for(int i = 1; i <= numAfactorial; i++){
        resultado = i*resultado;
    }
    cout<<"el factorial de "<<numAfactorial<<" es:"<<resultado;
    
    return 0;
}

File: Tareas/phyton/factorial/factorial.py
numFactorial = 0;
resultado = 1;
print("En este programa te daremos el factorial de un numero")
numFactorial = int(input("introduce el numero del cual quieras conocer el factorial"))
for i in range(1,numFactorial + 1):
    resultado = resultado * i
    
print("el factorial de ", numFactorial, " es: ", resultado)